src/main/java/com/foalex/bookstore/controller/BookController.java

```diff
@@ -17,35 +20,56 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for mapping books")
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("/api/books")
 public class BookController {
     private final BookService bookService;
 
+    @Operation(
+            summary = "Retrieves all books",
+            description = "Retrieves books with pagination and sorting by fields."
+    )
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    public List<BookDto> getAll(Pageable pageable) {

```

Comment:
it may be beneficial to use `@PageableDefault` in case client did not provide pagination parameters